weave

```diff
@@ -97,6 +99,32 @@ collect_cidr_args() {
     fi
 }
 
+is_ip() {
+    # The regexp here is far from precise, but good enough.
+    echo "$1" | grep -E '^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$' >/dev/null
```

Comment:
I thought about that as well... If you think its a good idea, I'll make the change